imagestreams/nodejs-centos7.json

```diff
@@ -0,0 +1,132 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "nodejs",
+    "annotations": {
+      "openshift.io/display-name": "Node.js"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
```

Comment:
We are currently at 10.x for `latest` https://github.com/bucharest-gold/centos7-s2i-nodejs/blob/master/imagestreams/nodejs-centos7.yml#L22

Reply:
I was planning on updating it at some point after the "10" tag was introduced but I think we can "go big or go home at this point". Concern was that "latest" currently follows "8" which is the 8 SCL image. Changing "latest" to follow "10" RHOAR image would be a big change. Note: we tell people that is not a good practice. Will submit update.